doc/user_manual/postprocessor.tex

```diff
@@ -1473,6 +1473,41 @@ \subsubsection{Interfaced}
 </Simulation>
 \end{lstlisting}
 
+\paragraph{Method: HistorySetDelay}
+
+This Post-Processor allows history sets to add delayed or lagged
+variables.  For example, if there a variable price that is set hourly,
+than new variable called price\_prev\_hour could be set by using a
+delay of -1.  This can be useful for training a ROM or other data
+analysis.
```

Comment:
Josh, please add an example here that precisely explains the Input DataObject and how the Output DataObject will get out. From the previous explanation I do not know how to use this PP and how to set up the output dataobject.
We need to be very verbose (with examples).

Reply:
Maybe in conjunction with the example you have below.

Reply:
Added a bit more here, and expanded the example to include the step and the data objects.